### ISSUE TITLE
Crash when testing " echo '今天什么股票涨？' | syntaxnet/models/parsey_universal/tokenize_zh.sh"！

### ISSUE THREAD

**Issue by 12343954** [tensorflow/models#1188]
## Please let us know which model this issue is about (specify the top-level directory)

`"echo 'I love you!' | syntaxnet/demo.sh"`, is OK

but, Crash with this，what can I do？somebody help me，thank you so mush！
`echo '今天什么股票涨？' | syntaxnet/models/parsey_universal/tokenize_zh.sh`

W external/org_tensorflow/tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W external/org_tensorflow/tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W external/org_tensorflow/tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W external/org_tensorflow/tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
F syntaxnet/term_frequency_map.cc:63] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: label-map)
syntaxnet/models/parsey_universal/tokenize_zh.sh: line 30: 89315 Abort trap: 6           $PARSER_EVAL --input=$INPUT_FORMAT --output=stdin-untoken --hidden_layer_sizes=256,256 --arg_prefix=brain_tokenizer_zh --graph_builder=structured --task_context=$CONTEXT --resource_dir=$MODEL_DIR --model_path=$MODEL_DIR/tokenizer-params --batch_size=1024 --alsologtostderr --slim_model

![wechatimg5](https://cloud.githubusercontent.com/assets/1804003/23989801/d7d2565c-0a6e-11e7-8e05-9d76cacc9eba.jpeg)

**Comment by winnerineast** [comment]
the error caused is "F syntaxnet/term_frequency_map.cc:63] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: label-map)
syntaxnet/models/parsey_universal/tokenize_zh.sh: line 30: 89315 Abort trap: 6 $PARSER_EVAL --input=$INPUT_FORMAT --output=stdin-untoken --hidden_layer_sizes=256,256 --arg_prefix=brain_tokenizer_zh --graph_builder=structured --task_context=$CONTEXT --resource_dir=$MODEL_DIR --model_path=$MODEL_DIR/tokenizer-params --batch_size=1024 --alsologtostderr --slim_model". And this model is trained based on traditional chinese instead of simplified chinese.

**Comment by djweiss** [comment]
That error means the path to resources are not set properly (it cannot find the files.) 

See instructions here: https://github.com/tensorflow/models/blob/master/syntaxnet/g3doc/universal.md

**Comment by 12343954** [comment]
I solved it!
use:
```
echo '然而,這樣的處理也衍生了一些問題.' | \
  syntaxnet/models/parsey_universal/tokenize_zh.sh $MODEL_DIRECTORY | \
  syntaxnet/models/parsey_universal/parse.sh $MODEL_DIRECTORY
```

thank you all !

see:
https://github.com/tensorflow/models/issues/334
